tests/functional/test_api.py

```diff
@@ -87,6 +101,34 @@ def user_with_token(user, db_session, factories):
 
 
 @pytest.fixture
+def auth_client(db_session, factories):
+    auth_client = factories.AuthClient(authority='thirdparty.example.org')
+    db_session.commit()
+    return auth_client
+
+
+@pytest.fixture
+def third_party_user(auth_client, db_session, factories):
+    user = factories.User(authority=auth_client.authority)
+    db_session.commit()
+    return user
+
+
+@pytest.fixture
+def publisher_group(auth_client, db_session, factories):
+    group = factories.PublisherGroup(authority=auth_client.authority)
```

Comment:
Hmm, groups _have_ to have a creator in the current model, yet none is specified here and this test passes.